src/scripts/main.js

```diff
@@ -18,10 +18,24 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+function createTree(container, data) {
+  const ul = document.createElement('ul');
 
-function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length) {
+      createTree(li, data[key]);
+    }
+
+    ul.appendChild(li);
+  }
+  ul.setAttribute('id', 'tree');
```

Comment:
Setting the `id` attribute for the `ul` element to a static value ('tree') could cause issues if `createTree` is called multiple times, as multiple elements will have the same `id`. Consider using a class or generating a unique `id` for each tree.